proto/privval/msgs.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+package tendermint.proto.privval;
+
+option go_package = "github.com/tendermint/tendermint/proto/privval";
+
+import "third_party/proto/gogoproto/gogo.proto";
+import "proto/crypto/keys/types.proto";
+import "proto/types/types.proto";
+
+message RemoteSignerError {
+  int64  code        = 1;
```

Comment:
should be uint8 tops

Reply:
size 32 is smallest in proto, in8, int16 and int32 are the same size in proto (1-2 bytes) just need to be careful on unmarshal and setting to handwritten type. we should not casttype to int8 like in consensus msgs as this can cause overflow on unmarshal. I will fix the consensus msgs in the consensus msgs pr